packages/runtime/src/useCases/transport/messages/SendMessage.ts

```diff
@@ -73,21 +74,53 @@ export class SendMessageUseCase extends UseCase<SendMessageRequest, MessageDTO>
             );
         }
 
-        if (!(transformedContent instanceof Request)) return;
+        if (transformedContent instanceof Notification) return;
+        if (!(transformedContent instanceof Notification)) {
+            const peerDeletedAddressArray: string[] = [];
+            const peerToBeDeletedAddressArray: string[] = [];
+            const missingOrInactiveRelationshipAddressArray: string[] = [];
+            const recipientsCoreAddress = recipients.map((r) => CoreAddress.from(r));
+            for (const recipient of recipientsCoreAddress) {
+                const relationship = await this.relationshipsController.getActiveRelationshipToIdentity(recipient);
+                if (!relationship) {
+                    missingOrInactiveRelationshipAddressArray.push(recipient.address);
```

Comment:
If you add a `continue` hereafter, you will be allowed to remove the ? after `relationship` in the following.